lib/bitex_bot/models/api_wrappers/kraken/kraken_api_wrapper.rb

```diff
@@ -6,96 +10,104 @@ def self.setup(settings)
     @settings = settings.kraken
   end
 
-  def self.transactions
-    client.public.trades('XBTUSD')[:XXBTZUSD].reverse.map { |t| transaction_parser(t) }
-  rescue NoMethodError => e
-    retry
-  end
-
-  def self.orders
-    KrakenOrder.open.map { |ko| order_parser(ko) }
+  def self.client
+    @client ||= KrakenClient.load(@settings)
   end
 
-  def self.order_book(retries = 20)
-    book = client.public.order_book('XBTUSD')[:XXBTZUSD]
-    order_book_parser(book)
-  rescue NoMethodError => e
-    retry
+  def self.amount_and_quantity(order_id, transactions)
+    KrakenOrder.amount_and_quantity(order_id, transactions)
```

Comment:
This will blow up, `KrakenOrder.amount_and_quantity` accepts a single argument.
See https://github.com/bitex-la/bitex-bot/pull/16/files#diff-04e5ef161c50760ad8e62a12aac1eb93R38